tests/SprykerSdkTest/Zed/AppSdk/Business/AppManifestFacadeTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Zed\AppSdk\Business;
+
+use Codeception\Test\Unit;
+
+/**
+ * @group SprykerSdkTest
+ * @group Zed
+ * @group AppSdk
+ * @group Business
+ * @group AppManifestFacadeTest
+ */
+class AppManifestFacadeTest extends Unit
+{
+    /**
+     * @var string
+     */
+    public const MESSAGE_NAME = 'MyMessage';
```

Comment:
Can be removed as not used here.

Please always look at the coverage output. We should not have unused code.